File: Leetcode/1304. Find N Unique Integers Sum up to Zero/solution.py
class Solution:
    def sumZero(self, n: int) -> List[int]:
        res = []
        loop = n//2 + 1
        rem = n % 2

        if rem == 1:
            res.append(0)
        
        for i in range(1, loop):
            res.append(i)
            res.append(-i)

        return res

File: Leetcode/1304. Find N Unique Integers Sum up to Zero/solution.go
func sumZero(n int) []int {
	var loop int = n/2 + 1
	rem := n % 2
	res := make([]int, 0)

	if rem == 1 {
		res = append(res, 0)
	}

	for i := 1; i < loop; i++ {
		res = append(res, i, -i)
	}

	return res
}